src/game/Objects/Object.cpp

```diff
@@ -2087,19 +2086,15 @@ void WorldObject::MovePositionToFirstCollision(Position& pos, float dist, float
 
     UpdateAllowedPositionZ(destX, destY, destZ);
     destZ += halfHeight;
-    bool colPoint = GetMap()->GetLosHitPosition(pos.x, pos.y, pos.z + halfHeight, destX, destY, destZ, -0.5f);
+    bool colPoint = GetMap()->GetLosHitPosition(pos.x, pos.y, pos.z + halfHeight, destX, destY, destZ, -1.0f);
```

Comment:
changed to -1, since we remove a -0.5 (contact_distance) distance modifier a few lines down.